cli/src/logger/mod.rs

```diff
@@ -5,12 +5,8 @@ pub enum ErrorType {
   GitChanges = 102,
   NoGit = 103,
   DeleteTmpAppTar = 104,
-  InvalidDefaultAnycloudAlias = 105,
-  DeployNotFound = 106,
   InvalidCredentialsFile = 108,
   InvalidAnycloudFile = 110,
-  InvalidDefaultCredentialAlias = 111,
-  InvalidCredentialAlias = 112,
```

Comment:
in the future we may want to use [#[deprecated]](https://doc.rust-lang.org/nightly/reference/attributes/diagnostics.html#the-deprecated-attribute) to ensure that we don't use any integer values that have been used before (although I don't think it's of concern now, when those error codes haven't even been used yet)

Reply:
thinking more about this, I think it actually makes sense to use a `u16` instead of a `u8` as the numeric representation of error codes. I don't imagine we'll ever surpass 65k error codes, but I think it's fairly likely we'll need more than 256 over the course of the next few years (keeping in mind such deprecations)

Reply:
I agree here. Except, let's be cheeky and use `u64` since that's all Alan cares about. ;)